source/dlangbot/github.d

```diff
@@ -338,7 +338,8 @@ private struct AllPages
     void popFront()
     {
         import std.utf : byCodeUnit;
-        url = link[1..$].byCodeUnit.until(">").array;
+        if (link)
```

Comment:
Should be `if (link.length)`, otherwise it only checks that the pointer is non-null.